package-system/OpenEXR/build_config.json

```diff
@@ -2,35 +2,64 @@
    "git_url":"https://github.com/AcademySoftwareFoundation/openexr.git",
    "git_tag":"v3.1.3",
    "package_name":"OpenEXR",
-   "package_version":"3.1.3-rev2",
+   "package_version":"3.1.3-rev3",
    "package_url":"https://www.openexr.com/",
    "package_license":"BSD-3-Clause",
    "package_license_file":"LICENSE.md",
    "cmake_find_source":"FindOpenEXR.cmake",
    "cmake_find_target":"FindOpenEXR.cmake",
    "extra_files_to_copy": [
-      ["FindImath.cmake", "FindImath.cmake"]
+      ["FindImath.cmake", "FindImath.cmake"],
+      ["../../Scripts/cmake/o3de_package_utilities.cmake", "o3de_package_utilities.cmake"]
+   ],
+   "cmake_install_filter":[
+      "*.cmake",
+      "*.a",
+      "*.lib",
+      "*.h",
+      "LICENSE.md"
+   ],
+   "cmake_generate_args_common" : [
+      "-DBUILD_TESTING=OFF",
+      "-DOPENEXR_BUILD_EXAMPLES=OFF",
+      "-DOPENEXR_INSTALL_EXAMPLES=OFF",
+      "-DOPENEXR_BUILD_TOOLS=OFF",
+      "-DOPENEXR_INSTALL_TOOLS=OFF",
+      "-DOPENEXR_RUN_FUZZ_TESTS=OFF",
+      "-DBUILD_SHARED_LIBS=OFF",
+      "-DDOCS=OFF",
+      "-DPYTHON=OFF"
    ],
    "Platforms":{
       "Windows":{
          "Windows":{
-            "build_configs" : ["debug", "release"],
+            "build_configs" : ["Debug", "Release"],
             "depends_on_packages" :[ 
                ["zlib-1.2.11-rev5-windows", "8847112429744eb11d92c44026fc5fc53caa4a06709382b5f13978f3c26c4cbd", ""]
             ],
             "cmake_generate_args":[
                "-G",
-               "\"Visual Studio 16 2019\"",
-               "-DBUILD_TESTING=OFF",
-               "-DOPENEXR_INSTALL_EXAMPLES=OFF",
-               "-DOPENEXR_BUILD_TOOLS=OFF",
-               "-DBUILD_SHARED_LIBS=OFF",
-               "-DCMAKE_INSTALL_PREFIX=\"../../openexr-windows/OpenEXR\""
+               "\"Visual Studio 16 2019\""
             ],
             "custom_test_cmd" : [
                "test_openexr_windows.cmd"
             ]
-         }
+         },
+         "Android": {
```

Comment:
It's strange that we have an Android and iOS OpenEXR package when the library is only used with host tool applications.

Reply:
I went to the bother of creating and testing those configs (and they work).  But yes, I coul dremove them?  But then thats just more work if we ever want to support them.
Thoughts?

Reply:
I think it is fine to keep it for now in the build config